README.md

```diff
@@ -1,3 +1,17 @@
 # docker-slack-message
 
 Very simple tool to send Slack messages. Built into a docker image
+
+## Testing commands
+
+```console
+export SLACK_CHANNEL=...
+export SLACK_TOKEN=...
+export IMAGE=...
+
+docker run -e SLACK_CHANNEL -e SLACK_TOKEN -e SLACK_MESSAGE=testingroot ${IMAGE}
+docker run -e SLACK_CHANNEL -e SLACK_TOKEN -e SLACK_MESSAGE=reply -e SLACK_THREAD_TS=<fill in from previous> ${IMAGE}
+docker run -e SLACK_CHANNEL -e SLACK_TOKEN -e SLACK_MESSAGE=reply2 -e SLACK_THREAD_TS=<fill in from previous> ${IMAGE}
+docker run -e SLACK_CHANNEL=<fill in from previous> -e SLACK_TOKEN -e SLACK_MESSAGE=reply-updated -e SLACK_UPDATE_MESSAGE_TS=<fill in from previous> ${IMAGE}
+docker run -e SLACK_CHANNEL=<fill in from previous> -e SLACK_TOKEN -e SLACK_DELETE_MESSAGE_TS=<fill in from previous> ${IMAGE}
```

Comment:
We could be a bit more helpful here in spelling out what you need to pass to do all the different operations. Put them under different subheadings or something.

Reply:
I turned it into a test script. It posts 4 messages, deletes one, edits one. Passes outputs from one step to the other